projectile.el

```diff
@@ -1261,10 +1261,11 @@ they are excluded from the results of this function."
   "Get files from sub-projects for PROJECT-ROOT recursively."
   (projectile-flatten
    (mapcar (lambda (sub-project)
-             (mapcar (lambda (file)
-                       (concat sub-project file))
+             (let ((project-relative-path (file-relative-name sub-project project-root)))
+               (mapcar (lambda (file)
+                       (concat (file-name-as-directory project-relative-path) file))
```

Comment:
I think you should move the `file-name-as-directory` to the `let`. Seems wasteful to call this within the `mapcar`.

Reply:
fixed in 748ed2b